src/mam4xx/mo_gas_phase_chemdr.hpp

```diff
@@ -273,8 +301,26 @@ void perform_atmospheric_chemistry_and_microphysics(
     // Dry deposition (gas)
     //----------------------
 
-    // FIXME: drydep integration in progress!
-    // mam4::drydep::drydep_xactive(...);
+    mam4::mo_drydep::drydep_xactive(
+        drydep_data,
+        fraction_landuse, // fraction of land use for column by land type
+        month,            // month
+        col_index_season, // column-specific mapping of month indices to
+                          // seasonal land-type indices [-]
+        sfc_temp,         // surface temperature [K]
+        air_temp,         // surface air temperature [K]
+        tv,               // potential temperature [K]
+        pressure_sfc,     // surface pressure [Pa]
+        pressure_10m,     // 10-meter pressure [Pa]
+        spec_hum,         // specific humidity [kg/kg]
+        wind_speed,       // 10-meter wind spped [m/s]
+        rain,             // rain content [??]
+        snow,             // snow height [m]
+        solar_flux,       // direct shortwave surface radiation [W/m^2]
+        mmr,              // constituent MMRs [kg/kg]
+        dvel,             // deposition velocity [1/cm/s]
+        dflx              // deposition flux [1/cm^2/s]
+    );
```

Comment:
need to add `if (kk == nlev)` for drydep_xactive. Only gas dry dep fluxes and velocity are only calculated at bottom layer. 

Some input vars need calculation. For example, wind_speed, it is calculated as sqrt(ubot^2 + vbot^2).